processor.py

```diff
@@ -32,7 +35,8 @@ def _clone_and_establish_baseline(self, event):
             logging.info(
                 'Cloning repository {} into {}'.
                 format(repo_full_name, repo_dir.name))
-            subprocess.run(["git", "clone", repo_url, repo_dir.name], stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+            repo_url_with_token = repo_url.replace("https://", "https://git:" + re.sub('[^0-9a-zA-Z]+', '', Config.access_token) + "@")
```

Comment:
I'm curious about this syntax: 

```
https://git:$TOKEN@repo
```

Everywhere in the docs I can find, I see that the username should be the user's username rather than "git". Do you have any docs that success this would work using the "git" username?

Reply:
would this work?
```
repo_url_with_token = repo_url.replace("https://", "https://{}:x-oauth-basic@".format(Config.access_token))
```

Reply:
i dont think the "git" part is idiomatic, it is just something i got in the habit of doing.  @itomaldonado has a much more elegant solution here.  https://github.blog/2012-09-21-easier-builds-and-deployments-using-git-over-https-and-oauth/ remarks that is idiomatic.